toolkit/imageconfigs/packagelists/azurevm-packages.json

```diff
@@ -4,6 +4,7 @@
         "cloud-init-azure-kvp",
         "cloud-utils-growpart",
         "dhcp-client",
+        "grub2",
```

Comment:
We want Azure images to be grub2-mkconfig-based by default, yes?

And core-efi remains legacy?